docs/advanced/dynamic-job-control.md

```diff
@@ -27,7 +27,10 @@ To add a job at runtime, leverage the `IRuntimeJobRegistry` interface:
 ```csharp
 app.MapPost("/add-job", (IRuntimeJobRegistry registry) => 
 {
-    var hasSucceeded = registry.TryRegister(n => n.AddJob<SampleJob>(p => p.WithCronExpression("* * * * *").WithName("MyName")), out Exception? exc);
+    var hasSucceeded = registry.TryRegister(
```

Comment:
Prevents an horizontal scrollbar on the doc.

![image](https://github.com/user-attachments/assets/a6ea5b74-4494-49d0-a231-f3de1c712a0d)